content/module-reference/utility-modules/shared/scopes.md

```diff
@@ -111,15 +111,21 @@ Bear in mind that these controls are provided as a basic guide to achieving colo
 
 # exposure adjustment
 
-The histogram, waveform, and RGB parade scopes can be used to directly alter the _exposure_ and _black level_ of the [_exposure_](../../processing-modules/exposure.md) module.
+The histogram, waveform, and RGB parade scopes can be used to directly alter the exposure and black level of the exposure module.
 
 For the histogram, click towards the right-hand side of the histogram and then drag right to increase or drag left to decrease the exposure. In a similar manner you can control the black level by clicking and dragging in the left-hand side.
 
 For horizontal waveform and RGB parade, drag the upper portion of the scope up/down to increase/decrease exposure. Drag the lower portion up/down to increase/decrease the black level.
 
 For vertical waveform and RGB parade scopes, the corresponding regions are to the right (exposure) and left (black level).
 
-You can also scroll in the appropriate area -- rather than dragging -- to adjust exposure and black point. Double-click in the scope to reset the exposure module's parameters to their defaults.
```

Comment:
Double-clicking to reset params is still valid in the modes waveform, rgb parade and histogram. I don't think it should be removed.